strategy/streaming.py

```
@@ -0,0 +1,557 @@
+                seq=self._seq,
+                server_time=int(time.time() * 1000),
+                data=data,
+            )
+        # Notify any registered async queues
+        if self._loop and self._loop.is_running():
+            msg = envelope.to_json()
+            for q in list(self._subscribers):
+                try:
+                    self._loop.call_soon_threadsafe(q.put_nowait, msg)
+                except Exception:
+                    pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)